internal/rbd/replicationcontrollerserver.go

```diff
@@ -336,9 +336,8 @@ func (rs *ReplicationServer) ResyncVolume(ctx context.Context,
 
 	mirroringInfo, err := rbdVol.getImageMirroringInfo()
 	if err != nil {
-		// TODO: check if the image is not found return valid error message
 		// in case of Resync the image will get deleted and gets recreated and
-		// it takes time for this operation
+		// it takes time for this operation.
```

Comment:
Should that be a `codes.Internal` error? Or, should it be `Aborted` so a retry can be done later?

Reply:
Aborted makes sense. Thanks for the suggestion.